runtime/common/src/constants.rs

```diff
@@ -350,7 +338,7 @@ pub type MaxPaginatedPageId = ConstU16<32>;
 /// The maximum number of actions in itemized actions
 pub type MaxItemizedActionsCount = ConstU32<5>;
 /// The number of blocks for Stateful mortality is 24 hours
```

Comment:
nit: This comment should be 48 hours to match the value below.

Reply:
Good point!